crates/ruff_linter/resources/test/fixtures/flake8_markupsafe/MS001.py

```diff
@@ -0,0 +1,13 @@
+import flask
+from markupsafe import Markup, escape
+
+content = "<script>alert('Hello, world!')</script>"
+Markup(f"unsafe {content}")  # MS001
+flask.Markup("unsafe {}".format(content))  # MS001
+Markup("safe {}").format(content)
+flask.Markup(b"safe {}", encoding='utf-8').format(content)
+escape(content)
+Markup(content)  # MS001
+flask.Markup("unsafe %s" % content)  # MS001
+Markup(object="safe")
+Markup(object="unsafe {}".format(content))  # Not currently detected
```

Comment:
Found one more example. In the ecosystem results we can find this example:

```python
Markup("*" * 8)
```

There is no user input here - essentially it's just string formatting -, so there is no possibility of tampering and we could allow these cases. 


https://github.com/apache/airflow/blob/340a70bfe7289e01898ddd75f8edfaf7772e9d09/airflow/www/views.py#L4523

Reply:
I agree, let's add these cases for when red-knot is available.